ipa-core/src/protocol/step.rs

```diff
@@ -43,8 +43,8 @@ pub enum DeadCodeStep {
     SaturatedSubtraction,
     #[step(child = crate::protocol::ipa_prf::prf_sharding::step::FeatureLabelDotProductStep)]
     FeatureLabelDotProduct,
-    #[step(child = crate::protocol::context::step::ZeroKnowledgeProofValidateStep)]
-    ZeroKnowledgeProofValidate,
+    #[step(child = crate::protocol::context::step::DZKPValidationStep)]
+    DZKPValidationStep,
```

Comment:
this does not follow naming principles for steps here. I don't like DZKP abbreviation, but if you want to stick with it here - it should be `DZKPValidate`